accord-core/src/main/java/accord/coordinate/CoordinateNoOp.java

```diff
@@ -76,13 +76,15 @@ void onPreAccepted(Topologies topologies, Timestamp executeAt, List<PreAcceptOk>
         }
         else
         {
-            new Propose<Timestamp>(node, topologies, Ballot.ZERO, txnId, txn, route, executeAt, Deps.NONE, this)
+            Deps deps = Deps.merge(successes, ok -> ok.deps);
+            new Propose<Timestamp>(node, topologies, Ballot.ZERO, txnId, txn, route, executeAt, deps, this)
             {
                 @Override
                 void onAccepted()
                 {
                     Writes writes = txn.execute(txnId, txnId, null);
                     Result result = txn.result(txnId, executeAt, null);
+                    Deps deps = Deps.merge(acceptOks, ok -> ok.deps);
                     Apply.sendMaximal(node, txnId, route, txn, executeAt, deps, writes, result);
```

Comment:
```suggestion
            Deps preacceptDeps = Deps.merge(successes, ok -> ok.deps);
            new Propose<Timestamp>(node, topologies, Ballot.ZERO, txnId, txn, route, executeAt, preacceptDeps, this)
            {
                @Override
                void onAccepted()
                {
                    Writes writes = txn.execute(txnId, txnId, null);
                    Result result = txn.result(txnId, executeAt, null);
                    Deps acceptDeps = Deps.merge(this.acceptOks, ok -> ok.deps);
                    Apply.sendMaximal(node, txnId, route, txn, executeAt, acceptDeps, writes, result);
```
Just a little more explicit about there being two different sets of deps in scope and that acceptOks comes from the `Propose`.